installer/dbscript/dbscript-2.sql

```diff
@@ -4610,3 +4610,9 @@ SET @auth_admin_verify_data_group_id := (SELECT `id` FROM ohrm_data_group WHERE
 
 INSERT INTO ohrm_user_role_data_group (`can_read`, `can_create`, `can_update`, `can_delete`, `self`, `data_group_id`,`user_role_id`)
 VALUES (1, 0, 0, 0, 0, @auth_admin_verify_data_group_id, @admin_role_id);
+
+ALTER TABLE `ohrm_timesheet_action_log` CHANGE `performed_by` `performed_by` INT(20) NULL DEFAULT NULL;
+
+ALTER TABLE `ohrm_timesheet_action_log` DROP FOREIGN KEY `ohrm_timesheet_action_log_ibfk_1`;
```

Comment:
check this `ohrm_timesheet_action_log_ibfk_1`

Reply:
the default fk generated seems to be the same even when I did a fresh install